index.html

```diff
@@ -1,4 +1,4 @@
----
+---samsung android 11
 layout: index
 lang: en
----
+docs/content/github/writing-on-github/working-with-advanced-formatting.md.
```

Comment:
I'm not sure, what are you exactly trying to do, but this kind of change is totally unnecessary here.

Reply:
@mutza974

Reply:
+4